apps/marketing/src/app/(marketing)/open/page.tsx

```diff
@@ -31,14 +31,13 @@ const ZStargazersLiveResponse = z.record(
 
 export type StargazersType = z.infer<typeof ZStargazersLiveResponse>;
 
-// const ZOpenPullRequestsResponse = ZMergedPullRequestsResponse;
-
 export default async function OpenPage() {
+  const randomQueryParam = `?${Math.random()}`;
   const {
     forks_count: forksCount,
     open_issues: openIssues,
     stargazers_count: stargazersCount,
-  } = await fetch('https://api.github.com/repos/documenso/documenso', {
+  } = await fetch(`https://api.github.com/repos/documenso/documenso${randomQueryParam}`, {
```

Comment:
You will notice above that we revalidate every hour meaning there will be a one hour delay between metrics. This is used for the card metrics at the top.

For the charts we use a separate API that polls GitHub every now and then to capture changes. The lag is fine for the most part since it's not super critical to have correct information at the second of the request.

Reply:
Ohhh, that makes sense, I had missed the revalidate param. I will go ahead and close this :)